tests/unit/components/pages/donation_form/Payment.spec.ts

```diff
@@ -8,17 +8,19 @@ import AmountField from '@src/components/shared/form_fields/AmountField.vue';
 import { nextTick } from 'vue';
 import RadioField from '@src/components/shared/form_fields/RadioField.vue';
 import { AddressTypeModel } from '@src/view_models/AddressTypeModel';
+import { DisplaySection, DisplaySectionCollection } from '@src/components/pages/donation_form/Payment';
 
 describe( 'Payment.vue', () => {
 	let store: Store<any>;
 
-	const getWrapper = (): VueWrapper<any> => {
+	const getWrapper = ( displaySections: DisplaySectionCollection = [ 'amount', 'interval', 'paymentType' ] ): VueWrapper<any> => {
```

Comment:
A handy way to avoid having to pass parameters to getWrapper is to set them manually in the test:

```js
const wrapper = getWrapper();
await wrapper.setProps( { displaySections: [ 'amount', 'interval', 'paymentType' ] } );
```